### ISSUE TITLE
cannot access AudioSystem error with 4.5-alpha-2

### ISSUE THREAD

**Issue by ZacSweers** [robolectric/robolectric#6046]
### Description

With Robolectric 4.5-alpha-2, `Shadows` fails to initialize when accessed due to using an `@hide` `AudioSystem` API.

```
/home/ubuntu/actions-runner/_work/slack-android-ng/slack-android-ng/app-legacy/src/test/java/com/Slack/ui/fileviewer/bottomsheet/FileViewerBottomSheetBinderTest.java:206: error: cannot access AudioSystem
    shadowOf(getMainLooper()).idle();
    ^
  class file for android.media.AudioSystem not found
```

### Steps to Reproduce

Try to use Shadows from a unit test on the given robolectric version

### Robolectric & Android Version

4.5-alpha-2, AGP 4.1, Gradle 6.7, Android 30

![image](https://user-images.githubusercontent.com/1361086/98304252-a4c29f00-1f8d-11eb-942f-95afd20b87d8.png)

![image](https://user-images.githubusercontent.com/1361086/98304264-ab511680-1f8d-11eb-9cf6-04f116489159.png)

![image](https://user-images.githubusercontent.com/1361086/98304280-b3a95180-1f8d-11eb-9a5f-5cedcab9e45e.png)

**Comment by hoisie** [comment]
This will be fixed with https://github.com/robolectric/robolectric/pull/6039

**Comment by hoisie** [comment]
Merged #6039
